packages/transport/src/modules/sync/externalEventProcessors/MessageDeliveredExternalEventProcessor.ts

```diff
@@ -1,11 +1,19 @@
+import { serialize, validate } from "@js-soft/ts-serval";
+import { CoreSerializable } from "../../../core/CoreSerializable";
 import { MessageDeliveredEvent } from "../../../events";
 import { Message } from "../../messages/local/Message";
 import { BackboneExternalEvent } from "../backbone/BackboneExternalEvent";
 import { ExternalEventProcessor } from "./ExternalEventProcessor";
 
+export class MessageDeliveredExternalEventData extends CoreSerializable {
```

Comment:
why is this exported? And why CoreSerializable instead of Serializable?

same for all other EventDatas

Reply:
I didnt know there is `Serializable`. What is the difference between `CoreSerializable` and `Serializable`

Reply:
I think CoreSerializable adds some functionalities on serval's Serializable. IDK where and if we even need that.